topi/python/topi/arm_cpu/depthwise_conv2d.py

```diff
@@ -134,7 +134,7 @@ def _callback(op):
                 data = data_pad.op.input_tensors[0]
             _schedule(cfg, s, data, data_pad, kernel, output)
 
-        if op.tag == 'spatial_depthwise_conv_nchw_output':
+        if op.tag == 'spatial_depthwise_conv2d_nchw_output':
```

Comment:
A test case to cover this would be very nice.

Reply:
agreed